package.json

```diff
@@ -81,6 +81,6 @@
   },
   "packageManager": "pnpm@8.3.1",
   "engines": {
-    "node": "^14.18.0 || ^16.10.0 || ^17.0.0 || ^18.0.0 || ^19.0.0"
+    "node": "^14.18.0 || ^16.10.0 || ^17.0.0 || ^18.0.0 || ^19.0.0 || ^20.0.0"
```

Comment:
What about adding this as `>=20.0.0` for forward compatibility?

Reply:
Hmm, wouldn't that mean that e.g. Node 21 would count as supported as well once it is released?
I remember https://github.com/nuxt/nuxt/pull/19687 and that you said only LTS versions should be supported officially if I remember correctly. That's why I chose to continue the caret notation.
Using `>=` would give the "benefit" of not having to create PRs like this for every new Node version, do I understand that correctly? Would that be desirable?

Reply:
It's more that I wouldn't want - in some future world where Node 36 has been released - for Nuxt 3.5 to be wholly unusable on it. I don't think it would count as an endorsement or 'supported' usage. I'm more interested in using the `engines.node` field to disallow _lower_ versions that have known issues with Nuxt.

Reply:
I see! So could we then shorten your suggestion to `"^14.18.0 || >=16.10.0"` or would I then be doing the semver we're speaking about wrong?

Reply:
Yep! Perfect. We can update the lower bound as needed.